pkg/operator/sync.go

```diff
@@ -206,6 +206,16 @@ func (optr *Operator) syncRenderConfig(_ *renderConfig) error {
 		spec.CloudProviderConfig = cc
 	}
 
+	if spec.Platform == "openstack" {
+		if infra.Spec.CloudConfig.Name != "" {
+			openstackCACert, err := optr.getCAsFromConfigMap("openshift-config", infra.Spec.CloudConfig.Name, "ca-bundle.pem")
+			if err != nil && !apierrors.IsNotFound(err) {
+				return err
+			}
+			spec.OpenStackCACertificate = openstackCACert
```

Comment:
What happens when the `ca-bundle.pem` key is not present in the config map, or when the config map isn't there?

Should this be enclosed in an else clause?